Bottom: 3856535c45f2f5e2ce535550e9fde9f596d71890
Top:    62f2eaba3f5e079d0c5f4f1797703bc11b4d0366
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 18:01:14 -0700

Refresh of iom-dbg

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 7f1839c..a26136d 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1458,12 +1458,8 @@ static int rs_send_iomaps(struct rsocket *rs, int flags)
 			break;
 	}
 
-<<<<<<< current
 	rs->iomap_pending = !dlist_empty(&rs->iomap_queue);
-=======
-	rs->iomap_pending = dlist_empty(&rs->iomap_queue);
 	printf("%s - pending = %d\n", __func__, rs->iomap_pending);
->>>>>>> patched
 	fastlock_release(&rs->iomap_lock);
 	return ret;
 }
